src/client/App.jsx

```diff
@@ -1,11 +1,11 @@
 import './App.css';
 
-import Registration from './components/Registration';
+import Login from './components/Login';
 
 export default function App() {
   return (
     <div className="App">
-      <Registration />
+      <Login />
```

Comment:
What happened to the Registration component?

Reply:
I only put the registration and app components in there for testing. Since I don't know yet how we're going to do the pages/routing, I removed the registration